components/BottomNavigation/README.md

```diff
@@ -181,3 +186,77 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+## Accessibility {#a11y}
```

Comment:
Please remove the duplicated heading. We can just use the one above.

Reply:
Done.